go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -205,6 +206,7 @@ type TabletConfig struct {
 	QueryCacheSize                int    `json:"queryCacheSize,omitempty"`
 	SchemaReloadIntervalSeconds   int    `json:"schemaReloadIntervalSeconds,omitempty"`
 	WatchReplication              bool   `json:"watchReplication,omitempty"`
+	TrackSchemaVersions           bool   `json:"trackSchemaVersions,omitempty"`
```

Comment:
We should set this default value to `true`. People should disable this only if they encounter problems with this.